cypress/page/shopping-cart.page.ts

```diff
@@ -0,0 +1,13 @@
+class ShoppingCartPage {
+  private checkoutButton: string;
+
+  constructor() {
+    this.checkoutButton = ".cart_navigation span";
+  }
+
+  public proceedToCheckoutII(): void {
+    cy.get(this.checkoutButton).click();
+  }
```

Comment:
There's no need to call as "II" because this is form ShoppingCart, it's not sharing the method or implementation with product list page
```suggestion
  public proceedToCheckout(): void {
    cy.get(this.checkoutButton).click();
  }
```